bitnami/kafka/templates/client-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.provisioning.enabled }}
```

Comment:
This file has to be removed, since client.properties will now be created during the container initialization.